crates/typos/src/tokens.rs

```diff
@@ -401,8 +408,8 @@ mod parser {
         <T as Stream>::Token: AsChar + Copy,
     {
         (
-            take_while1(is_localport_char),
-            opt(preceded(':', take_while0(is_localport_char))),
+            take_while(1.., is_localport_char),
+            opt(preceded(':', take_while(.., is_localport_char))),
```

Comment:
nit: could you use `0..` instead of `..`?

They would mean the same thing but this will cause the compiler to only instantiate one version of the generic function rather than two.

Reply:
(unsure if it'd actually make a difference)